### ISSUE TITLE
Multi-Label Support

### ISSUE THREAD

**Issue by thiagosantos1** [helboukkouri/character-bert#9]
It would be nice if Multi-Label Support is added.

**Comment by helboukkouri** [comment]
Hi @thiagosantos1 :) Sorry for the delay. What do you mean by Multi-Label support ?

**Comment by thiagosantos1** [comment]
@helboukkouri At this time, seems like only binary and multi-class classification is supported. Maybe it would be good to add a multi-label support as well ( an input can be assigned to multiple labels).

I was looking at the code, and it doesn't seems hard to make the modification. Basically gotta change the loss function from "Categorical Cross Entropy" to "Binary Cross Entropy”

**Comment by helboukkouri** [comment]
Okay I see. So as you say it's pretty easy to adapt the existing code for multi-label classification. However, this repo is not really meant to offer an exhaustive list of finetuning scenarios but rather a very basic example of CharacterBERT usage so that people can get started with the pre-trained models 😊 In any case, the model will become part of huggingface's `transformers` in the future so that any finetuning code that works with BERT can be seamlessly re-used by just swaping a few classes. More info about the integration [->here<-](https://github.com/helboukkouri/character-bert/issues/6)
